src/main/java/io/gatling/demostore/security/WebsiteSecurityConfig.java

```diff
@@ -35,6 +36,9 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
+        http
+                .sessionManagement()
+                .sessionCreationPolicy(SessionCreationPolicy.STATELESS); // Avoid creating session in memory
```

Comment:
From the documentation:

> Spring Security will never create an HttpSession and it will never use it to obtain the SecurityContext

(source: [SessionCreationPolicy API](https://docs.spring.io/spring-security/site/docs/4.2.5.RELEASE/apidocs/org/springframework/security/config/http/SessionCreationPolicy.html)


So I guess that Spring Security won't create it.
I do hope my session repository will be able to create it even if `getSession(false)` (the argument is "create a new session") but create it from the cookie